packages/labs/compiler/README.md

```diff
@@ -1,6 +1,93 @@
 # @lit-labs/compiler
 
-> **Warning**
-> Work in progress, not ready for external use.
+A compiler for optimizing Lit templates.
 
-Tracking RFC: https://github.com/lit/rfcs/pull/21
+> **Warning** `@lit-labs/compiler` is part of the Lit Labs set of packages – it is published
+> in order to get feedback on the design and may receive breaking changes.
+> Tracking RFC: https://github.com/lit/rfcs/pull/21
+
+> **Warning** `@lit-labs/compiler` is not yet published so it cannot be installed from npm.
+
+## Overview
+
+`@lit-labs/compiler` exports a [TypeScript
+Transformer](https://github.com/itsdouges/typescript-transformer-handbook#types-of-transformers)
```

Comment:
Wouldn't this be a better intro to answering what a typescript transformer actually is?
https://github.com/itsdouges/typescript-transformer-handbook#the-basics

```suggestion
Transformer](https://github.com/itsdouges/typescript-transformer-handbook#the-basics)
```

Reply:
very much yes. Applied!